cEP-2.md

```diff
@@ -0,0 +1,33 @@
+#Next Gen Bear Design
+| Metadata     | 					                        |
+| ------------ |-----------------------------------------|
+| cEP          | 2                                       |
+| Version      | 2.1                                     |  
```

Comment:
nope, this is cEP-0002 which is the name, it's version is 0.1 at proposal, 1.0 whn accepted. Please change status to proposed and set version to .1